### ISSUE TITLE
Failed to build magma/klee/libpng

### ISSUE THREAD

**Issue by exKAZUu** [HexHive/magma#56]
I've tried to evaluate `klee` with magma and libpng, but I got the following error:

```
[2021-01-28 07:39] Building magma/klee/libpng
[2021-01-28 07:47] Failed to build magma/klee/libpng. Check build log for info.
```

The build log is as follows:

```
...
-- Found klee-uclibc library: "/magma/fuzzers/klee/uclibc/lib/libc.a"
-- klee-libcxx support enabled
-- Use libc++ include path: "/magma/fuzzers/klee/libcxx/libc++-install-9/include/c++/v1"
^[[91mCMake Error at CMakeLists.txt:579 (message):
   does not exist. Try passing -DKLEE_LIBCXXABI_SRC_DIR=<path>

^[[0m-- Configuring incomplete, errors occurred!
See also "/magma/fuzzers/klee/klee/build/CMakeFiles/CMakeOutput.log".
See also "/magma/fuzzers/klee/klee/build/CMakeFiles/CMakeError.log".
```

The complete reproducible steps are as follows:

1. Install VirtualBox on Mac OS
2. `vagrant init generic/ubuntu1804` (so I'm using Ubuntu 18.04 on VirtualBox)
3. `vagrant up --provider=virtualbox`
4. `ssh -p 2222 vagrant@localhost`
5. Entere the Ubuntu terminal on VirtualBox
6. `sudo apt-get update && sudo apt-get install -y util-linux inotify-tools docker.io git`
7. `sudo gpasswd -a $USER docker`
8. `sudo systemctl restart docker`
9. `git clone https://github.com/HexHive/magma.git`
10. `git log`
```
commit 5d81041f3d9154ade9f0fe889fb65f3f77329571 (HEAD -> v1.1, origin/v1.1, origin/HEAD)
Author: Ahmad Hazimeh <ahmad+gh@hazimeh.dev>
Date:   Thu Dec 17 17:44:47 2020 +0100

    Anchored sqlite3 to proper working reference
```   
11. `cd magma/tools/captain`
12. Configure `captainrc` as follows:
```
REPEAT=1
TIMEOUT=1m
FUZZERS=(aflplusplus honggfuzz angora klee)
aflplusplus_TARGETS=(libpng)
honggfuzz_TARGETS=(libpng)
angora_TARGETS=(libpng)
klee_TARGETS=(libpng)
```
13. `./run.sh`

**Comment by exKAZUu** [comment]
The current klee requires `libcxx` ... ? c.f. https://github.com/klee/klee/pull/966

**Comment by yh570** [comment]
I met the same problem here. Magma failed to build all provided targets with KLEE.

**Comment by adrianherrera** [comment]
Hey folks, please check out https://github.com/HexHive/magma/pull/60

This should fix the build issues

**Comment by hazimeh** [comment]
Fixed in #60
